docs/rules/padding-line-between-tags.md

```diff
@@ -0,0 +1,128 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/padding-line-between-tags
+description: Require or disallow newlines between sibling tags in template
+---
+# vue/padding-line-between-tags
+
+> Require or disallow newlines between sibling tags in template
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule requires or disallows newlines between sibling HTML tags
+
+<eslint-code-block fix :rules="{'vue/padding-line-between-tags': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>
+    <div></div>
+
+    <div>
+    </div>
+
+    <div />
+  </div>
+</template>
+```
+
+```vue
+<!-- ✗ BAD -->
+<template>
+  <div>
+    <div></div>
+    <div>
+    </div>
+    <div />
+  </div>
+</template>
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+```json
+{
+  "vue/padding-line-between-tags": ["error", "always" | "never" , {
+      "ignoreNewlinesBefore": [],
+      "ignoreNewlinesAfter": []
+  }]
+}
+```
+
+- `always` (default) Requires one or more blank lines.
+- `never` Disallows blank lines between tags
+- `ignoreNewlinesBefore` ignores newlines before specified elements. Only applies when using `always`.
+    default `[]`
+- `ignoreNewlinesAfter` ignores newlines after specified elements. Only applies when using `always`.
+    default `[]`
```

Comment:
Hmm. With the current options, I don't think a rule can both always force blank lines and always disallow blank lines.
I think it would be better to refer to the padding-line-between-statements rule option and make it possible to set "never" and "always" in more detail.

https://eslint.org/docs/latest/rules/padding-line-between-statements

For example:

```js
: [
        "error",
        { "blankLine": "always", "prev": "*", "next": "*" },
        { "blankLine": "never", "prev": "<br>", "next": "*" },
        { "blankLine": "never", "prev": "*", "next": "<br>" },
        // ...
    ]
```